.circleci/config.yml

```diff
@@ -108,7 +109,34 @@ jobs:
             printf "%% FilterByTag\n" >> tests/filteredTest/TaggedTest.m
             printf "classdef (TestTags = {'FILTERED'}) TaggedTest < matlab.unittest.TestCase\n" >> tests/filteredTest/TaggedTest.m
             printf "methods (Test)\n function testTag (testCase)\n assert(2==2);\n end\n end\n end" >> tests/filteredTest/TaggedTest.m
+            mkdir simtests
+            cat \<<'_EOF' >> "simtests/createModelAndTest.m"
+            model = 'simple_model';
+            evalin('base','bdclose all');
+            if exist('simple_model.slx', 'file') == 0
+                new_system(model);
+                load_system(model);
+                add_block('built-in/Constant', [model, '/Constant']);
+                save_system(model);
+                close_system(model);
+                sltest.testmanager.clear;
+                sltest.testmanager.clearResults;
+                tf = sltest.testmanager.TestFile('test.mldatx');
+                cs = tf.getCoverageSettings;
+                cs.RecordCoverage = true;
+                cs.MdlRefCoverage = true;
+                cs.MetricSettings = 'd';
+                ts = tf.getTestSuites;
+                tc = ts.getTestCases;
+                tc.setProperty('model', model);
+                tf.saveToFile;
+                tf.close;
+                sltest.testmanager.close;
+            end
+            disp('Created Model and Simulink Test file to simulate the model.');
+            _EOF
```

Comment:
This setup logic is getting sophisticated enough where I think we should begin to consider pre-creating these files and checking them in.

I think this is OK for now though.

Reply:
I agree, probably we could commit these files as part of next set of changes.